packages/wonder-blocks-layout/util/util.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import type {MediaQuery, MediaSize} from "./types.js";
+
+export const queryMatchesSize = (
+    mediaQuery: MediaQuery,
+    mediaSize: MediaSize,
+): boolean => {
+    switch (mediaQuery) {
+        case "all":
+            return true;
+        case "small":
+            return mediaSize === "small";
+        case "mdOrSmaller":
+            return mediaSize === "medium" || mediaSize === "small";
+        case "medium":
+            return mediaSize === "medium";
+        case "mdOrLarger":
+            return mediaSize === "medium" || mediaSize === "large";
+        case "large":
+            return mediaSize === "large";
+        default:
+            throw new Error(`Unsupported mediaSize: ${mediaSize}`);
+    }
```

Comment:
NIT: I'm used to having a newline after each case. Not sure if that's a standard style of ours or not.

Reply:
There isn't anything in our style guides, I can add newlines.